prepare-videochat.sh

```diff
@@ -338,7 +347,7 @@ fi
 if lsmod | grep v4l2loopback >/dev/null 2>/dev/null; then
     # module is already loaded, do nothing
     :
-else
+elif [ $CAPTURE_STREAM = v -o $CAPTURE_STREAM = av ]; then
```

Comment:
Audio+Video is default: https://github.com/laptander/ipwebcam-gst/blob/master/prepare-videochat.sh#L158
But in this elif statement I check if it is v or av. And (as "av" is written in 158 line), it loads module by default. I think "-o" is "logical or" operator.